### PR TITLE
Update to dev versions

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
I misunderstood how we were going to version tools packages post RTM. I'll send an update in a bit. Hold off on reviewing this.

**Comment by pranavkm** [comment]
Updated

**Comment by natemcmaster** [comment]
Missed some. These test projects still reference "1.0.0"

- test/Microsoft.EntityFrameworkCore.InMemory.Tests/project.json
- test/Microsoft.EntityFrameworkCore.Tools.FunctionalTests/project.json
- test/Microsoft.EntityFrameworkCore.Tools.Core.Tests/project.json
- test/Microsoft.EntityFrameworkCore.Sqlite.Tests/project.json
- test/Microsoft.EntityFrameworkCore.Sqlite.Design.Tests/project.json
- test/Microsoft.EntityFrameworkCore.SqlServer.Tests/project.json
- test/Microsoft.EntityFrameworkCore.SqlServer.Design.Tests/project.json
- test/Microsoft.EntityFrameworkCore.Tools.Tests/project.json

**Comment by pranavkm** [comment]
Those don't reference * versions and consequently already rely on being project references. Feel free to fix them in a follow up.

**Comment by natemcmaster** [comment]
> Those don't reference * versions and consequently already rely on being project references.

Won't this still break when we run tests without the src folder? Or is that something we aren't supporting anymore?

File: test/Microsoft.EntityFrameworkCore.Sqlite.FunctionalTests/project.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
   },
   "dependencies": {
     "dotnet-test-xunit": "1.0.0-*",
-    "Microsoft.EntityFrameworkCore.Relational.Specification.Tests": "1.0.0-*",
-    "Microsoft.EntityFrameworkCore.Sqlite": "1.0.0-*"
+    "Microsoft.EntityFrameworkCore.Relational.Specification.Tests": "1.1.0-*",
```
Comment (pranavkm) [inline test/Microsoft.EntityFrameworkCore.Sqlite.FunctionalTests/project.json]: The src project references are fine. Test -> test is the one that's broken.

**Pull request closed without merging**